js/src/components/account-card/index.js

```diff
@@ -212,6 +215,11 @@ export default function AccountCard( {
 							{ indicator }
 						</div>
 					) }
+					{ errorSlots && (
```

Comment:
@ankitrox I think `errorSlots` will always be defined. So no need to have a check here.